integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/cache/CarbonDropCacheCommand.scala

```diff
@@ -55,13 +55,12 @@ case class CarbonDropCacheCommand(tableIdentifier: TableIdentifier, internalCall
         carbonTable.getTableName)) {
         DataMapUtil.executeClearDataMapJob(carbonTable, DataMapUtil.DISTRIBUTED_JOB_NAME)
       } else {
-        val allIndexFiles = CacheUtil.getAllIndexFiles(carbonTable)(sparkSession)
         // Extract dictionary keys for the table and create cache keys from those
         val dictKeys: List[String] = CacheUtil.getAllDictCacheKeys(carbonTable)
-
         // Remove elements from cache
-        val keysToRemove = allIndexFiles ++ dictKeys
-        cache.removeAll(keysToRemove.asJava)
+        cache.removeAll(dictKeys.asJava)
+        DataMapStoreManager.getInstance()
```

Comment:
nit: `clearDataMaps` method has another variant with just `AbsoluteTableIdentifier` as argument. Probably for consistency, we used that method whenever the second argument(`launchJob` is true). Like below -
`DataMapStoreManager.getInstance().clearDataMaps(carbonTable.getAbsoluteTableIdentifier)`